scipy/signal/ltisys.py

```diff
@@ -26,6 +26,7 @@
 # np.linalg.qr fails on some tests with LinAlgError: zgeqrf returns -7
 # use scipy's qr until this is solved
 
+import six
```

Comment:
This should be from `scipy._lib.six`, no? (And same for the `tests` code presumably.)